core/src/main/java/de/danoeh/antennapod/core/util/NetworkUtils.java

```diff
@@ -66,6 +66,18 @@ public static boolean autodownloadNetworkAvailable() {
 						}
 					}
 				}
+			} else {
+				if (!UserPreferences.isEnableAutodownloadOnMobile())
+				{
```

Comment:
Please move the brackets to the same line ;) Just to keep the same code style all over the project